galaxy_importer/main.py

```diff
@@ -90,15 +90,18 @@ def call_importer(filepath, cfg):
     namespace, name, version = match.groups()
     filename = collection.CollectionFilename(namespace, name, version)
 
-    with open(filepath, 'rb') as f:
-        try:
-            data = collection.import_collection(f, filename, cfg=cfg)
-        except ImporterError as e:
-            logging.error(f'The import failed for the following reason: {str(e)}')
-            return None
-        except Exception:
-            logging.error('Unexpected error occurred:', exc_info=True)
-            return None
+    try:
+        data = collection.import_collection(
+            file=None,
+            filepath=filepath,
+            filename=filename,
+            cfg=cfg)
+    except ImporterError as e:
+        logging.error(f'The import failed for the following reason: {str(e)}')
+        return None
+    except Exception:
+        logging.error('Unexpected error occurred:', exc_info=True)
```

Comment:
You can setup a module scope logger in this module and use it here. Just using logging.error will use the root / unnamed logger which makes it harder tell where it is coming from.  For humans and for log processing tools. 

ie,
``` python
log = logging.getLogger(__name__)
# or if __name__ ends up __main__ (not sure what happens when entrypoints are used), then an
# explicit
# log = logging.getLogger('galaxy_importer.main')
...
     except Exception:
         log.exception(''Unexpected error occurred:')
```

log.exception() is equiv to log.error('some message', exc_info=True)